docs/use/commands/pmm-admin.md

```diff
@@ -448,12 +455,15 @@ Enable all collectors and limit monitoring for `dbstats`, `indexstats`, `collsta
 
 PMM collects metrics in two [resolutions](../../how-to/configure.md#metrics-resolution) to decrease CPU and Memory usage: high and low resolutions.
 
-In high resolution we collect metrics from collectors which work fast:
+In high resolution we collect metrics from collectors that work fast:
+
 - `diagnosticdata`
 - `replicasetstatus`
 - `topmetrics`
+- `currentop`
```

Comment:
@BupycHuk is currentop metrics getting with high resolution?

Reply:
no, it's low resolution